appveyor.yml

```diff
@@ -5,10 +5,20 @@ platform: Any CPU
 shallow_clone: true
 clone_folder: c:\Cosmos
 
+
+
 init:
 - cmd: |
     git clone https://github.com/CosmosOS/IL2CPU.git c:\IL2CPU --depth 1
 
+install:
+  - ps: |
+      $range = "$env:APPVEYOR_REPO_BRANCH..$env:APPVEYOR_PULL_REQUEST_HEAD_COMMIT"
+      if (-not (git diff --name-only --diff-filter=d $range | Where-Object { $_ -NotMatch "\.md$" })) {
```

Comment:
Can we extend this to include more files from the skip list below?

Reply:
We could, but in as I replied to the other changes requested, I think we need to fix GitHub status checks before we worry about adding more excludes :)

Reply:
AppVeyor exits the build based on the git diff, so the list can now be extended 👍🏽